config/download.json

```
@@ -1,4 +1,17 @@
 {
+  "3.1.9": {
+    "time": "2023-12-25",
+    "src": {
+      "src": "https://www.apache.org/dyn/closer.lua/dolphinscheduler/3.1.8/apache-dolphinscheduler-3.1.9-src.tar.gz",
+      "asc": "https://downloads.apache.org/dolphinscheduler/3.1.8/apache-dolphinscheduler-3.1.9-src.tar.gz.asc",
+      "sha512": "https://downloads.apache.org/dolphinscheduler/3.1.8/apache-dolphinscheduler-3.1.9-src.tar.gz.sha512"
+    },
+    "bin": {
+      "src": "https://www.apache.org/dyn/closer.lua/dolphinscheduler/3.1.8/apache-dolphinscheduler-3.1.9-bin.tar.gz",
```

Seem we add error path here.
```suggestion
      "src": "https://www.apache.org/dyn/closer.lua/dolphinscheduler/3.1.9/apache-dolphinscheduler-3.1.9-bin.tar.gz",
```

---

done.